.github/workflows/lint.yml

```diff
@@ -22,3 +22,4 @@ jobs:
     - run: flake8
     # To avoid datagetter errors, we must specifically exclude src/datagetter
     - run: black --check --exclude "/(\.eggs|\.git|_cache|src\/datagetter)/" ./
```

Comment:
Maybe a better way would be not to install all the other dependencies and just select the lint packages. e.g.

```yaml
- run: pip install `/bin/grep "black==" ./requirements_dev.txt`
- run: black . --check
```

Should be a bit faster too.

Reply:
A good general point, but this applied before and it applies to many of our lint runs so I'd request it's for a different pull request.

Reply:
Ah didn't notice this wasn't part of the PR, my mistake.